docs/architecture/grpc-for-wcf-developers/service-mesh.md

```diff
@@ -14,46 +14,40 @@ A service mesh is an infrastructure component that takes control of routing serv
+The following screenshot shows the StockWeb application running with the Linkerd service mesh. There are no changes to the application code, and the Docker image isn't being used. The only change required was the addition of an annotation to the deployment in the YAML files for the `stockdata` and `stockweb` services.
 
-![StockWeb with Service Mesh](media/service-mesh/stockweb-servicemesh-screenshot.png)
+![StockWeb with service mesh](media/service-mesh/stockweb-servicemesh-screenshot.png)
 
-You can see from the Server column that the requests from the StockWeb application have been routed to both replicas of the StockData service, despite originating from a single `HttpClient` instance in the application code. In fact, if you review the code, you'll see that all 100 requests to the StockData service are made simultaneously using the same `HttpClient` instance, yet with the service mesh, those requests will be balanced across however many service instances are available.
+You can see from the **Server** column that the requests from the StockWeb application have been routed to both replicas of the StockData service, despite originating from a single `HttpClient` instance in the application code. In fact, if you review the code, you'll see that all 100 requests to the StockData service are made simultaneously by using the same `HttpClient` instance. With the service mesh, those requests will be balanced across however many service instances are available.
 
-Service meshes only apply to traffic within a cluster. For external clients, see [the next chapter, Load Balancing](load-balancing.md).
+Service meshes apply only to traffic within a cluster. For external clients, see the next chapter, [Load Balancing](load-balancing.md).
 
 ## Service mesh options
 
-There are three general-purpose service mesh implementations currently available for use with Kubernetes: Istio, Linkerd, and Consul Connect. All three provide request routing/proxying, traffic encryption, resilience, host-to-host authentication, and traffic control.
+Three general-purpose service mesh implementations are currently available for use with Kubernetes: [Istio](https://istio.io), [Linkerd](https://linkerd.io), and [Consul Connect](https://consul.io/mesh.html). All three provide request routing/proxying, traffic encryption, resilience, host-to-host authentication, and traffic control.
 
-Choosing a service mesh depends multiple factors:
+Choosing a service mesh depends on multiple factors:
 
-- The organization's specific requirements around costs, compliance, paid support plans, and so on.
-- The nature of the cluster, its size, the number of services deployed, and the volume of traffic within the cluster network.
-- Ease of deploying and managing the mesh and using it with services.
+- The organization's specific requirements around costs, compliance, paid support plans, and so on
```

Comment:
These should have punctuation: https://docs.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation 

```suggestion
- The organization's specific requirements around costs, compliance, paid support plans, and so on.
```

Reply:
I followed the style guideline and left off punctuation when all the items in a list were fragments or didn't finish the intro sentence. But adding it to those items is something readers aren't likely to notice.